### PR TITLE
Final compliance changes

### PR DESCRIPTION
**Pull request by mskinner5278**

### CODE REVIEW

File: utils/healthcheck.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def check_working_directory():
         check_working_directory.remediation = f'Please start LSDC in {daq_utils.beamline} data directory. Current directory: {working_dir}'
         return False
     if daq_utils.getBlConfig("visitDirectory") != os.getcwd():
-        check_working_directory.remediation = ("Working directory mismatch. Please start LSDC GUI in the same folder as the server is running.")
```
Comment (JunAishima) [inline utils/healthcheck.py:78]: This line was constructed in the original commit in such a way that the users could not have the possibility of seeing any metadata that might be encoded in the other user's directory name.

**Review by JunAishima — COMMENTED** [review]
comment about the visit directory checking procedure showing which directory/proposal it should be.

File: daq_main2.py [hunk 2]
```diff
@@ -28,12 +28,12 @@
 logging.getLogger('ophyd').setLevel(logging.WARN)
 logging.getLogger('caproto').setLevel(logging.WARN)
 handler1 = handlers.RotatingFileHandler('lsdcServerLog.txt', maxBytes=5000000, backupCount=100)
-handler2 = handlers.RotatingFileHandler('/var/log/dama/%slsdcServerLog.txt' % os.environ['BEAMLINE_ID'], maxBytes=5000000, backupCount=100)
+#handler2 = handlers.RotatingFileHandler('/var/log/dama/%slsdcServerLog.txt' % os.environ['BEAMLINE_ID'], maxBytes=5000000, backupCount=100)
```
Comment (tacaswell) [inline daq_main2.py:31]: delete stuff you want to remove rather than leaving it commented out.

File: daq_macros.py [hunk 3]
```diff
@@ -1849,31 +1852,27 @@ def snakeRasterBluesky(rasterReqID, grain=""):
...
-    logger.info(f"Governor move to DA and synchronous arming took {time.time() - start_time} seconds.")
-    if govStatus.exception():
-      logger.error(f"Problem during start-of-raster governor move, aborting! exception: {govStatus.exception()}")
-      return
+                       num_images_per_file=numsteps) # rasterDef['numCells']) TODO: try to get all images in one file
+    #raster_flyer.detector.stage()
     procFlag = int(getBlConfig("rasterProcessFlag"))
     spotFindThreadList = []
-    yield from bps.mv(samplexyz.omega, (omega-1)) # attempting to over-compensate omega movement
+    yield from bps.mv(samplexyz.omega, (omega-1)) # attempting to over-compensate omega movemen
```
Comment (tacaswell) [inline daq_macros.py:1875]: added a spelling error?

File: daq_main2.py (hunk 2 above)
Comment (mskinner5278) [reply daq_main2.py]: fixed with [b2befa6](https://github.com/NSLS-II/lsdc/pull/315/commits/b2befa6f2f83776ff0ac107083f1f4a2df6223dc)

File: daq_macros.py (hunk 3 above)
Comment (mskinner5278) [reply daq_macros.py]: [cba1f53](https://github.com/NSLS-II/lsdc/pull/315/commits/cba1f53d8ce74be2eb0f484ef300cdbad5ac2de1)

File: utils/healthcheck.py (hunk 1 above)
Comment (tacaswell) [reply utils/healthcheck.py:78]: I think this needs to be addressed.

Comment (mskinner5278) [reply utils/healthcheck.py:78]: reverted with [0ebc395](https://github.com/NSLS-II/lsdc/pull/315/commits/0ebc3954d2a355a2a284b3e3b4e6345b6286b981)

**Review by tacaswell — APPROVED** [review]

**Pull request merged**